rbc/unicodetype_db.h

```diff
@@ -0,0 +1,46 @@
+#include <stdint.h>
+#include "path/to/numba/_unicodetype_db.h"
+
+// To generate unicodetype_db.ll, fix the include above and compile it with
+//      clang -S -emit-llvm -O2 unicodetype_db.h -o unicodetype_db.ll
```

Comment:
To avoid unnecessary changes to this file, I suggest:
```suggestion
#include "_unicodetype_db.h"

// To generate unicodetype_db.ll, fix the include above and compile it with
//      clang -S -emit-llvm -O2 -Ipath/to/numba/ unicodetype_db.h -o unicodetype_db.ll
```